fastlane/Fastfile

```diff
@@ -78,7 +83,9 @@ platform :ios do
     device_hash = {}
     device_hash[device_name] = device_udid
     register_devices(devices: device_hash)
-    match(force: true)
+    
+    match(type: "development", force: true)
+    match(type: "adhoc", force: true)
```

Comment:
I think we can improve functions in MatchManager to support this, for example:
```rb
  def sync_development_signing(app_identifier:, force: false)
    if @is_ci
      create_ci_keychain
      @fastlane.match(
        type: 'development',
        keychain_name: @keychain_name,
        keychain_password: @keychain_password,
        app_identifier: app_identifier,
        readonly: !force,
        force: force
      )
    else
      @fastlane.match(type: 'development', app_identifier: app_identifier, readonly: !force, force: force)
    end
  end
```

Same with the function `sync_adhoc_signing`.

Then you can use:
```rb
match_manager.sync_development_signing(app_identifier: [...], force: true)
match_manager.sync_adhoc_signing(app_identifier: [...], force: true)
```

Reply:
@markgravity Did you update this?

Reply:
@suho my bad, pushed them now 🙇